outputs.tf

```
@@ -0,0 +1,19 @@
+output "resource_group_name" {
+  description = "The resource group name"
+  value       = module.resource_group.name
+}
+
+output "resource_id" {
+  description = "The resource id of the resource group"
+  value       = module.resource_group.resource_id
+}
+
+output "user_assigned_managed_identities_resource_id" {
```

Plural name should be singular?
```suggestion
output "user_assigned_managed_identity_resource_id" {
```